lib/generation/generators/value_objects/file_structure_strategy/command_invoker.dart

```diff
@@ -0,0 +1,5 @@
+import 'package:parabeac_core/generation/generators/value_objects/file_structure_strategy/commands/file_structure_command.dart';
+
+abstract class CommandInvoker {
```

Comment:
I would suggest just placing a small description on what we are using this abstract class for :thinking:

Reply:
Good observation. Will add descriptions to this and the other abstract classes from this PR.